test-suite/src/test/scala/org/scalajs/testsuite/javalib/ExpectExceptions.scala

```diff
@@ -0,0 +1,21 @@
+package org.scalajs.testsuite.javalib
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.reflect.ClassTag
+
+trait ExpectExceptions extends JasmineTest {
+
+  final def expectNoException(expr: => Any): Unit =
+    expect(expr.##).not.toThrow
```

Comment:
I don't understand why `##`. Besides, you're missing `() =>` to make it a JS function. I think this should be:
```scala
expect(() => expr).not.toThrow
```
But then again, why not use that instead of `expectNoException` in the first place?